default.nix

```diff
@@ -104,6 +104,9 @@ let
     in
       args: pkgs.callPackage ./scripts/launch/connect-to-cluster (args // { inherit gitrev; } // walletConfig );
   other = rec {
+    demoCluster = pkgs.callPackage ./scripts/launch/demo-cluster { inherit gitrev; };
+    walletIntegrationTests = pkgs.callPackage ./scripts/test/wallet/integration { inherit gitrev; };
+    buildWalletIntegrationTests = pkgs.callPackage ./scripts/test/wallet/integration/build-test.nix { inherit walletIntegrationTests pkgs; };
```

Comment:
i'm not totally sure, but is it necessary to pass `pkgs` into `pkgs.callPackage`? Could jq and curl become args of `build-test.nix`?

Reply:
Yes, if they're arguments, it will pull the nixpkgs version, not the pinned versions we expect. We could hard code pkgs, like it is in this file and that would have the same affect. Was trying to keep this derivation as simple as possible though.